tests/unittest_modutils.py

```diff
@@ -439,5 +447,14 @@ def test_is_module_name_part_of_extension_package_whitelist_success(self) -> Non
         )
 
 
+@pytest.mark.skipif(not HAS_URLLIB3, reason="This test requires urllib3.")
+def test_file_info_from_modpath__SixMetaPathImporter() -> None:
+    pytest.raises(
+        ImportError,
```

Comment:
So we no longer have a finder that will find the module created by `_SixMetaPathImporter`, but that's better than crashing while trying to find it with `ExplicitNamespacePackageFinder` (or incorrectly finding it with it!)